src/_posts/databases/postgresql/2000-01-01-accessing.md

```diff
@@ -0,0 +1,238 @@
+2. From the command line, use our `dbclient-fetcher` helper to download and
+   install a bunch of PostgreSQL® tools:
+   ```bash
+   dbclient-fetcher pgsql
+   ```
+   The output should look like this:
+   ```bash
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 14.6
+   ```
+3. If you ever need a specific version, just add it as a second parameter:
+   ```bash
+   dbclient-fetcher pgsql 13
+   ```
+   The output should look like this:
+   ```bash
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 13.11
+   ```
+
+The helper installs the following tools:
+- `psql`
+- `pg_basebackup`
+- `pg_controldata`
+- `pg_dump`
+- `pg_isready`
+- `pg_recvlogical`
+- `pg_restore`
+- `pg_test_fsync`
+- `pg_upgrade`
+- `pg_archivecleanup`
+- `pg_config`
+- `pg_ctl`
+- `pg_dumpall`
+- `pg_receivewal`
+- `pg_resetwal`
+- `pg_rewind`
+- `pg_test_timing`
+- `pg_waldump`
+
+
+## Using Third Party Tools
+
+By default, and for security reasons, your Scalingo for PostgreSQL® database is
+not directly accessible from the Internet.
+
+To access your database remotely you first need to [make it reachable over the
+Internet](#making-the-database-reachable-over-internet):
+- either locally, on your computer, via an [encrypted tunnel](#setting-up-an-encrypted-tunnel)
+- or from any location, by [enforcing TLS connection]({% post_url databases/postgresql/2000-01-01-connecting %}#enforcing-tls-connection)
+  and [enabling direct Internet access](#enabling-direct-access-over-internet).
+
+Once a secured connection has been established, you should be able to connect
+to your database with your tool of choice. If you don't have one yet, we
+suggest you to [take a look at pgAdmin](#using-pgadmin).
+
+###  Making the Database Reachable Over Internet
+
+#### Using Our Command Line Tool
+
+Our Command Line tool provides a `db-tunnel` command that creates an SSH
+encrypted tunnel, allowing remote access to your Scalingo for PostgreSQL®
+addon.
+
+This tunnel does not provide any additional security layer. We only verify that
+your public key is registered on our platform. It acts as an encrypted bridge
+between the computer running the command and our infrastructure.
+
+1. Make sure you have correctly [setup the Scalingo Command Line tool]({% post_url platform/cli/2000-01-01-start %})
+2. Make sure you have correctly [setup SSH authentication]({% post_url platform/getting-started/2000-01-01-first-steps %})
+   on your Scalingo account
+3. From the command line, open the tunnel using this command:
+   ```bash
+   scalingo --app my-app db-tunnel SCALINGO_POSTGRESQL_URL
+   ```
+   The output should look like this:
+   ```bash
+   Building tunnel to my-app-4553.postgresql.a.osc-fr1.scalingo-dbs.com:33712
+   You can access your database on:
+   127.0.0.1:10000
+   ```
+4. As stated, the database is now reachable from `127.0.0.1` on port `10000`
```

Comment:
issue: Writing "As stated" kind of suggests to the reader that if they don't remember it they may not be smart enough. I would remove these words as they don't add anything.

```suggestion
4. The database is now reachable from `127.0.0.1` on port `10000`
```